tests/core/string_conversion.sy

```diff
@@ -59,6 +59,8 @@ start :: fn {
     print(d)
 
     // Other
-    print(A, pop, :str)
+    print' A
+    print' pop
+    print' :str
     as_str(nil) <=> "nil"
```

Comment:
This makes me sad...

Reply:
Not being able to print all at the same time? We could assume that they will look a specific way. Or just do a as_str instead.

Reply:
This is equivalent to what was there before - these changes should be done in a different pr.